### DISCUSSION TITLE
Exclude Rowing?  (Maybe related to #559)

### DISCUSSION THREAD

**Discussion by fix-digital** [philosowaffle/peloton-to-garmin#586] [Ideas]
Sometimes I use the Peloton App on a tablet with a Concept2 rowing machine, which I also use for other non-Peloton workouts.  Concept2 already syncs to Garmin/Strava and provides more data than Peloton, so seems like it would make sense to exclude here.  

Full disclosure, I've only been using your wonderful tool for about 12 hours, so it's possible Garmin's duplicate detection already handles this, but seems like it would make sense to exclude on principle.

**Comment by philosowaffle** [comment]
Seems the documentation is a little out of date, `Rowing` should already work as a valid value for the [ExcludeWorkoutTypes](https://philosowaffle.github.io/peloton-to-garmin/v4.0.0/configuration/json/#peloton-config) config.

The full list that I need to update on the docs site should be:

```
  Cycling
  OutdoorCylcing *added in 4.1.0-rc
  BikeBootcamp
  TreadmillRunning
  OutdoorRunning
  TreadmillWalking
  OutdoorWalking
  Cardio
  Circuit
  Strength
  Stretching
  Yoga
  Meditation
  Rowing
```

**Reply by fix-digital** [reply]
Got it, thanks!
